src/components/Cards/styled.ts

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+interface Props {
+  isLoading: boolean;
+}
+
+export const CardsWrapper = styled.div<Props>`
```

Comment:
it is also legal to do like that if there are only 1 prop, and declaration stay as 1 line
```suggestion
export const CardsWrapper = styled.div<{isLoading: boolean}>`
```

Reply:
ok